extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -1837,5 +1840,40 @@ describes.realWin(
         ).to.be.false;
       });
     });
+
+    describe('page loading', () => {
```

Comment:
this carries 0 context for a developer running the test suite and breaking something